README.md

```diff
@@ -3,3 +3,27 @@
 To develop install `jekyll` and `ruby-kramdown` and start the jekyll server with `./serve.sh`
 
 To just start a live server without local dependencies via Docker run `./docker.sh`
+
+## Notes
+
+### Convert Markdown to HTML
+
+We use [Lynx](https://lynx.invisible-island.net/) to convert Markdown to HTML. It creates a text file with the HTML content and adds the links as footnotes.
+
+You can either install `lynx` on your system or use the Docker image `alpine/lynx`.
+
+```bash
+docker run --rm -it alpine/lynx --dump URL > output.txt
+```
+
+Example for release notes:
+
+```bash
+docker run --rm -it alpine/lynx --dump https://relax-and-recover.org/documentation/release-notes-2-8 > release-notes-2-8.txt
```

Comment:
This does not ensure in general that
the generated release-notes-2-8.txt is ASCII, cf.
https://github.com/rear/rear/wiki/Coding-Style#character-encoding

E.g. on my openSUSE Leap 15.5 workstation
```
$ lynx --dump https://relax-and-recover.org/documentation/release-notes-2-8 >release-notes-2-8.txt

$ file release-notes-2-8.txt
release-notes-2-8.txt: UTF-8 Unicode text

$ locale
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```

To ensure the generated release-notes-2-8.txt is ASCII
one must run lynx in POSIX locale:
```
$ export LC_ALL=POSIX
$ export LANG=POSIX

$ lynx --dump https://relax-and-recover.org/documentation/release-notes-2-8 >release-notes-2-8.txt

$ file release-notes-2-8.txt
release-notes-2-8.txt: ASCII text
```

Reply:
By the way:
Also with lynx the generated release-notes-2-8.txt
has the unwanted header and footer like
```
   [1]Relax-and-Recover
     * [2]Home
     * [3]Features
     * [4]Documentation
     * [5]Downloads
     * [6]Support
     * [7]Development
     * [8]Events

```
and
```
   [20]
   
  If you like Relax-and-Recover please consider sponsorship
```
with the current online
https://relax-and-recover.org/documentation/release-notes-2-8
(i.e. without the changes in this pull request).